docs/developer-docs/latest/development/plugins/i18n.md

```diff
@@ -0,0 +1,377 @@
+---
+title: Internationalization (i18n) - Strapi Developers Documentation
+description: Instructions on how to use Strapi Content API with the Internationalization (i18n) optional plugin
+---
+
+# Internationalization (i18n)
+<!-- ? wondering if it would be a good idea to add the 🌍 icon to the h1 — if yes, we should do this for the h1 of other plugins as well -->
+
+> Internationalization is the design and development of a product, application or document content that enables easy localization for target audiences that vary in culture, region, or language.
+>
+> -- <cite>[W3C](https://www.w3.org/International/questions/qa-i18n.en#i18n)</cite>
+
+<!-- ? not sure if using a citation is a good idea to introduce/define i18n — feedback is welcome! -->
+
+The Internationalization (i18n) plugin allows Strapi users to create, manage and distribute content in several languages.
```

Comment:
For this, we should probably mirror the npm/yarn commands to install as well.

Reply:
+1